roles/ngi-rnaseq/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+- name: Download & install NextFlow 
+  shell: "curl -fsSL get.nextflow.io | bash"
+  args:
+    chmod: "{{ root_path }}/sw"
```

Comment:
`chdir`, not `chmod`. 

What do you think about putting the binary in a subdir instead, and not directly under `sw/`?